### PR TITLE
deps: Update to usvg 0.43.0

### PR DESCRIPTION
**Pull request by Eoghanmc22**

### CODE REVIEW

**Review by simbleau — APPROVED** [review]
Thank you! Can you add this to the changelog as well?

**Comment by Eoghanmc22** [comment]
Added, Not sure why the pr reference isn't rendering as a link in the preview or if that's just because it hasn't been merged. I turned on allow maintainer edits so you can fix any formatting issues b4 merging.

File: Cargo.toml [hunk 1]
```diff
@@ -36,7 +36,7 @@ workspace = true
 [dependencies]
 vello = { workspace = true }
 thiserror = "1.0.61"
-usvg = "0.42.0"
+usvg = "0.43.0"
 image = { version = "0.25.0", default-features = false, features = [
```
Comment (waywardmonkeys) [inline Cargo.toml:40]: I think you need to add `webp` to the features for the `image` crate as well.

**Review by waywardmonkeys — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (Eoghanmc22) [reply Cargo.toml:40]: good catch

**Comment by waywardmonkeys** [comment]
> Added, Not sure why the pr reference isn't rendering as a link in the preview or if that's just because it hasn't been merged. I turned on allow maintainer edits so you can fix any formatting issues b4 merging.

This is because you need to add a `[#31]: ...` line towards the bottom like the other PRs have.

**Pull request merged**